static/app/views/issueDetails/groupEventCarousel.tsx

```diff
@@ -378,6 +487,9 @@ const EventId = styled('span')`
       display: flex;
     }
   }
+  @media (max-width: 600px) {
```

Comment:
custom breakpoint?

Reply:
Our smallest breakpoint is 800 😞  - I think this is necessary for the best experience